neptune/experiments.py

```diff
@@ -603,6 +603,39 @@ def log_artifact(self, artifact, destination=None):
                           upload_tar_api_fun=self._backend.extract_experiment_output,
                           experiment=self)
 
+    def delete_artifacts(self, path):
+        """Removes an artifact(s) (file/directory) from the experiment storage.
+
+        Args:
+            path (:obj:`list`): List of paths to remove from the experiment's output
+
+        Raises:
+            `FileNotFound`: If a path in experiment artifacts does not exist.
+
+        Examples:
+            Assuming that `experiment` is an instance of :class:`~neptune.experiments.Experiment`.
+
+            .. code:: python3
+
+                experiment.delete_artifacts('forest_results.pkl')
+                experiment.delete_artifacts('')
+        """
+        if path is not None:
+            paths = path
+            if not isinstance(path, list):
+                paths = [path]
+            for path in paths:
+                if path.startswith(".."):
+                    raise ValueError("path to delete must be within project's directory")
```

Comment:
Remove this whole condition and add
```
path = os.path.abspath(os.sep + path.strip(os.sep))[1:]
```

Reply:
I disagree. Your way changes path that goes to server to output-relative and may cause user dot incidentally delete a file he might not have wanted

Reply:
Could you give me an example? I don't see such possibility.

Reply:
passing `somthing to '../file'` would delete `/file` in artifact output, which is confusing and in most cases wrong. I added this check as a simple pre-send validation of a request that would fail anyway due to server

Reply:
Yeah. This is exactly what executing `rm ../tmp/file` from root directroy would do - remove `/tmp/file`. Ask @kamil-kaczmarek

Reply:
as agreed, we'll use os.path.normpath to check if path starts with `..`